src/lilURL.php

```diff
@@ -236,6 +246,10 @@ public function isSafeURL($url)
         return true;
     }
 
+    protected function validateGAData($utmData){
```

Comment:
Please correct the conditional code style by adding a space so `($utmData){` is `($utmData) {`